demo/admin/src/links/Link.tsx

```diff
@@ -57,18 +57,8 @@ export const Link: DocumentInterface<Pick<GQLLink, "content">, GQLLinkInput> & D
     menuIcon: LinkIcon,
     ...createDocumentRootBlocksMethods(rootBlocks),
     ...createDocumentDependencyMethods({
+        rootQueryName: "link",
         rootBlocks,
-        query: gql`
-            query LinkDependency($id: ID!) {
-                node: link(linkId: $id) {
-                    id
-                    content
-                    pageTreeNode {
-                        id
-                        category
-                    }
-                }
-            }
-        `,
+        basePath: ({ pageTreeNode }) => `/pages/pagetree/${categoryToUrlParam(pageTreeNode.category)}/${pageTreeNode.id}/edit`,
```

Comment:
I think that's an okay amount of config

The only potential downside is that people might not realise what they are actually configuring and where it's used. But that's a frequent problem. Also I started adding docs in https://github.com/vivid-planet/comet/pull/1804 and will extend them further once we've decided on this